### ISSUE TITLE
[V2] sideMenu gesture doesn't work after set visible to false

### ISSUE THREAD

**Issue by manelsanz** [wix/react-native-navigation#4772]
### Issue Description
In order to hide `sideMenu`, there are the `visible` option. When it's set to `false`, if the `sideMenu` is showing it hides at the moment. But, when you try the swipe gesture to invoke it again, that appears for a sec and hides automatically.

### Steps to Reproduce
Set `visible: false` with `Navigation.mergeOptions()` method and try to do gesture from the left edge of screen to right.

### Workaround
The only workaround I found is call two times `Navigation.mergeOptions()`, one with `enabled: false` and other with `enabled: true`, but this is a bit weird I think.

```
  toggleOffSideMenu = () => {
    Navigation.mergeOptions(this.props.componentId, {
      sideMenu: {
        left: {
          enabled: false
        }
      }
    });
     Navigation.mergeOptions(this.props.componentId, {
       sideMenu: {
         left: {
           enabled: true
         }
       }
     });
  };
```

---
### Environment
* React Native Navigation version: 2.7.1
* React Native version: 0.57.8
* Platform(s): Android tested
* Device info: Xiaomi Mi 8 Lite (Android 8.1) / Huawei MediaPad T5 (Android 8.0)

**Comment by guyca** [comment]
This should be fixed in `2.12.0-snapshot.211`

**Comment by davidnum** [comment]
Does not fixed yet, 2.19.0
